src/modules/UI/scenes/TransactionList/TransactionList.ui.js

```diff
@@ -265,77 +179,71 @@ export class TransactionList extends Component<Props, State> {
         <Gradient style={[styles.currentBalanceBox]}>
-          {this.state.balanceBoxVisible && (
-            <Animated.View style={[styles.balanceBoxContents, { opacity: this.state.balanceBoxOpacity }]}>
-              <TouchableOpacity onPress={this.toggleShowBalance} style={[styles.currentBalanceWrap]}>
-                {showBalance ? (
-                  <View style={styles.balanceShownContainer}>
-                    <View style={[styles.iconWrap]}>
-                      {logo ? (
-                        <Image style={[{ height: '100%' }]} source={{ uri: logo }} resizeMode={'cover'} />
-                      ) : (
-                        <T style={[styles.request]}>{displayDenomination.symbol}</T>
-                      )}
-                    </View>
-                    <View style={[styles.currentBalanceBoxBitsWrap]}>
-                      <View style={{ flexDirection: 'row' }}>
-                        {displayDenomination.symbol ? (
-                          <T numberOfLines={1} style={[styles.currentBalanceBoxBits, styles.symbol]}>
-                            {displayDenomination.symbol + ' '}
-                            <T numberOfLines={1}>{cryptoAmountString}</T>
-                          </T>
-                        ) : (
-                          <T numberOfLines={1} style={styles.currentBalanceBoxBits}>
-                            {cryptoAmountString}
-                          </T>
-                        )}
+          <View style={styles.balanceBoxContents}>
+            {!isAccountBalanceVisible ? (
+              <View style={[styles.totalBalanceWrap]}>
+                <View style={[styles.hiddenBalanceBoxDollarsWrap]}>
+                  <T style={[styles.currentBalanceBoxHiddenText]}>{SHOW_BALANCE_TEXT}</T>
+                </View>
+              </View>
+            ) : (
+              <View style={styles.balanceShownContainer}>
+                <View style={[styles.iconWrap]}>
+                  {logo ? (
+                    <Image style={[{ height: '100%' }]} source={{ uri: logo }} resizeMode={'cover'} />
+                  ) : (
+                    <T style={[styles.request]}>{displayDenomination.symbol}</T>
+                  )}
+                </View>
+                <View style={[styles.currentBalanceBoxBitsWrap]}>
+                  <View style={{ flexDirection: 'row' }}>
+                    {displayDenomination.symbol ? (
+                      <T numberOfLines={1} style={[styles.currentBalanceBoxBits, styles.symbol]}>
+                        {displayDenomination.symbol + ' '}
+                        <T numberOfLines={1}>{cryptoAmountString}</T>
+                      </T>
+                    ) : (
+                      <T numberOfLines={1} style={styles.currentBalanceBoxBits}>
+                        {cryptoAmountString}
+                      </T>
+                    )}
```

Comment:
what does this change do?

Reply:
This is the implementation of the **actual** "Show Balance" version of the box, which was **not** there before. It was present on the WalletList but not this TransactionList